moochub-schema.json

```diff
@@ -1,3177 +1,3188 @@
-                                      },
-                                      "contentUrl": {
-                                        "type": [
-                                          "string",
-                                          "null"
-                                        ],
-                                        "format": "iri",
-                                        "example": null,
-                                        "description": "A link to the specific license document (if any, otherwise set to \"null\" - the identifier pointing at the general license is to be considered the license document then). Has to be \"null\" if identifier is \"proprietary\"."
-                                      }
-                                    },
-                                    "required": [
-                                      "identifier",
-                                      "url"
-                                    ]
-                                  }
+                                    "type": "array",
+                                    "description": "A list of licenses that apply for this resource. General licenses according to https://spdx.org/licenses/ (in the \"identifier\" and \"url\" attribute) or \"proprietary\" can be given. Additionally, it is possible to link to an optional, specific, own license document in the \"contentUrl\" attribute.",
+                                    "uniqueItems": true,
+                                    "minItems": 1,
+                                    "items": {
+                                        "title": "License",
+                                        "type": "object",
+                                        "description": "This schema describes the attributes of a license.",
+                                        "properties": {
+                                            "identifier": {
+                                                "description": "A license shortcode according to https://spdx.org/licenses/ or \"proprietary\".",
+                                                "example": "CC-BY-SA-4.0",
+                                                "type": "string"
+                                            },
+                                            "url": {
+                                                "type": [
+                                                    "string",
+                                                    "null"
+                                                ],
+                                                "format": "iri",
+                                                "description": "A license according to https://spdx.org/licenses/ or \"null\" if proprietary.",
+                                                "example": "https://creativecommons.org/publicdomain/zero/1.0"
+                                            },
```

Comment:
Here, for example, I see another issue: The example given doesn't seem to be consistent. 

- The `identifier` is given as `CC-BY-SA-4.0`, but the `url` is `https://creativecommons.org/publicdomain/zero/1.0`.
- The `url` is syntactically valid, but doesn't match to the "description", where a license in accordance with <https://spdx.org/licenses/> is requested.

Reply:
fixed